CHANGES.rst

```diff
@@ -1,13 +1,18 @@
 Changelog of fews-3di
 ===================================================
 
-
-2.3 (unreleased)
+2.4 (unreleased)
 ----------------
 
 - Nothing changed yet.
 
 
+2.3 (2023-02-22)
+----------------
+
+- Update usage of API token instead of username/password.
```

Comment:
```suggestion
- We now use an API token instead of a deprecated username/password 
  login when connecting to 3Di. You need to change your `settings.xml`:
  remove `username` and `password` and add `api_key`, with an api key
  you generated on the 3Di website.
```

Reply:
Goede, aangepast.